src/foundry/foundry.js/roll.d.ts

```diff
@@ -375,15 +378,15 @@ declare class Roll<D extends object = {}> {
    * @returns A promise which resolves to the created ChatMessage entity, if create is true
    *          or the Object of prepared chatData otherwise.
    */
-  toMessage<T extends DeepPartial<ChatMessage.CreateData> = {}>(
+  toMessage<T extends DeepPartial<ConstructorParameters<typeof ChatMessage>[0]> = {}>(
```

Comment:
Has this be to replaced with anything like `ConfiguredDocumentClass<typeof ChatMessage>`?

Reply:
Yeah I think that would probably need to be the configured ChatMessage class. The same goes for the return value.